pkgs/development/compilers/llvm/10/compiler-rt/default.nix

```
@@ -25,7 +25,7 @@ stdenv.mkDerivation {
     "-DCOMPILER_RT_DEFAULT_TARGET_ONLY=ON"
     "-DCMAKE_C_COMPILER_TARGET=${stdenv.hostPlatform.config}"
     "-DCMAKE_ASM_COMPILER_TARGET=${stdenv.hostPlatform.config}"
-  ] ++ lib.optionals (haveLibc && !isMusl) [
+  ] ++ lib.optionals (haveLibc && stdenv.hostPlatform.libc == "glibc") [
```

```suggestion
  ] ++ lib.optionals (haveLibc && stdenv.hostPlatform.isGnu) [
```
I think we could be able to use this here